clippy_dev/src/lintcheck.rs

```diff
@@ -198,18 +221,21 @@ impl Crate {
 
         let shared_target_dir = clippy_project_root().join("target/lintcheck/shared_target_dir/");
 
+        let mut args = vec!["--", "--message-format=json", "--", "--cap-lints=warn"];
+
+        if let Some(options) = &self.options {
+            for opt in options {
+                args.push(opt);
+            }
+        } else {
```

Comment:
... which is why I am thinking we should either mention this in the readme explicitly or add something like `-Aclippy::all` here to turn off default lints..?

Reply:
Since using Clippy with `-Aclippy::all` is a bit beside the point for this repo, I'd rather mention this in the README.